db/migrate/20200625215657_remove_introduction_text_from_interactive_page.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveIntroductionTextFromInteractivePage < ActiveRecord::Migration
+  def up
+    remove_column :interactive_pages, :text
```

Comment:
There is also the 'show_introduction` column that I think should be removed here.